src/prompt.rs

```diff
@@ -56,6 +56,38 @@ pub trait UserInteraction {
     fn select(&self, key: &str, values: &[Value]) -> Result<String>;
 }
 
+pub fn substitute_graphql_interactive<I>(
```

Comment:
This is very similar to the `substitute_interactive` except for which method they are calling, so I can combine them somehow